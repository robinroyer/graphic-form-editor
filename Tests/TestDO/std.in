C c01 10 10 4
R r01 7 7 14 14
L l01 6 6 12 13
PL pl01 1 5 9 14
S s01 0 0 20 20
LIST
MOVE c01 5 5
MOVE r01 5 5
MOVE l01 5 5
MOVE pl01 5 5
LIST
MOVE s01 3 3
LIST
MOVE c01 -15 -19
MOVE r01 -42 -21
MOVE l01 -46 -23
MOVE pl01 -27 -31
LIST
MOVE s01 -1 -2
LIST
EXIT