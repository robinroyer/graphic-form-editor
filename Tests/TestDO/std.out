OK
#New object: c01
OK
#New object: r01
OK
#New object: l01
OK
#New object: pl01
OK
#New object: s01
C c01 10 10 4
L l01 6 6 12 13
L pl01 1 5 9 14
R r01 7 7 14 14
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
C c01 15 15 4
L l01 11 11 17 18
L pl01 6 10 14 19
R r01 12 12 19 19
OK
#Object sucessfully moved
C c01 18 18 4
L l01 14 14 20 21
L pl01 9 13 17 22
R r01 15 15 22 22
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
C c01 3 -1 4
L l01 -32 -9 -26 -2
L pl01 -18 -18 -10 -9
R r01 -27 -6 -20 1
OK
#Object sucessfully moved
C c01 2 -3 4
L l01 -33 -11 -27 -4
L pl01 -19 -20 -11 -11
R r01 -28 -8 -21 -1
